Entities/Character/Character.gd

```diff
@@ -88,7 +88,7 @@ func got_shot(dmg, source):
 		hp = 0
 		being_removed = true
 		GameManager.world.gameplay_info.rpc("update_kills_info", str(source), self.get_name())
```

Comment:
You have to also add "!= "Heart attack" in update_players_info because game crashes after exiting to lobby